scripts/travis_script.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+# make sure we have the typings
+ls typings/codemirror || exit 1
```

Comment:
this line is causing travis to fail - is this because we can't get the typings on travis from PRs due to tokens? This may be something we need to reconsider because all PRs will fail. Maybe we just check in the typings and get rid of the tsd dependency...